src/chains.jl

```diff
@@ -95,6 +104,11 @@ function namesingroup(chains::Chains, sym::Symbol)
     return names_of_params[indices]
 end
 
+"""
+    group(chains::Chains, name::Union{String,Symbol}
+
+Returns a subset of the chain chain with all parameters in the group `:name`.
```

Comment:
The same applies here (and below):

```suggestion
Return a subset of the chain chain with all parameters in the group `:name`.
```

Reply:
In the next commit, I have run a grep over the repo to solve these kinds of issues.